tests/Feature/WorkerToolingTest.php

```
@@ -15,250 +13,231 @@
-            'site_id' => $loadBalancer->id,
-            'user' => 'isolated-foo',
-            'command' => 'node app.js',
-            'status' => WorkerStatus::RUNNING,
-        ]);
-
-        app(RefreshSiteWorkerConfigs::class)->refresh($this->isolatedSite, 'node');
-
-        $fake->assertNotExecutedContains("/etc/supervisor/conf.d/{$lbWorker->id}.conf");
-        $this->addToAssertionCount(1);
-    }
+    $ref = new ReflectionProperty($fake, 'uploadedContent');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '138', column '16'). (undefined)

(MissingImport)